crates/cxx-qt-gen/src/parser/qenum.rs

```diff
@@ -54,12 +56,20 @@ impl ParsedQEnum {
             ));
         }
 
+        let namespace = attribute_find_path(&qenum.attrs, &["namespace"])
+            .map(|attr_index| {
+                let attr = &qenum.attrs[attr_index];
+                expr_to_string(&attr.meta.require_name_value()?.value)
+            })
+            .transpose()?
+            .unwrap_or_default();
+
         // TODO: Add support for `namespace`, `cxx_name` and `rust_name` attributes.
```

Comment:
namespace has been added now? so can be removed from the comment

Reply:
Done